admin/services/business-availability.service.js

```diff
@@ -45,4 +48,23 @@ businessAvailabilityService.determinePinGenerationEligibility = (isLiveCheck, ch
   }
 }
 
+/**
+ * Retuns data for the avalibilty partial
+ * @param {Object} checkWindowData
+ * @returns {Promise<any>}
+ */
+businessAvailabilityService.getAvailabilityData = async (dfeNumber, checkWindowData) => {
+  const currentDate = moment.utc()
+  const hdfSubmitted = await headteacherDeclarationService.isHdfSubmittedForCheck(dfeNumber, checkWindowData.id)
+  const checkWindowClosed = currentDate.isAfter(checkWindowData.checkEndDate)
```

Comment:
I believe the final day should be admin end date and NOT check end date.
HDF can be submitted after check end date which should be an edge case.

Reply:
Hmmmm, the requirements specifically state: “After the check end date and before the admin end date, school staff can view read-only access arrangements page, as per prototype”